cypress/support/fragments/data_import/job_profiles/jobProfileEdit.js

```diff
@@ -17,4 +17,15 @@ export default {
       .click();
     cy.do(Modal({ id: 'unlink-job-profile-modal' }).find(Button('Unlink')).click());
   },
+  unlinkMatchProfile: (number) => {
+    cy.get('[id*="branch-ROOT-MATCH-editable"]').eq(number).find('button[icon="unlink"]').click();
```

Comment:
Please use Interactors

Reply:
what interactor i can use for this element [id*="branch-ROOT-MATCH-editable"]?

Reply:
There is no Interactor for such element for a while. But for button use Button interactor (find('button[icon="unlink"])

Reply:
can you use interactors in cy.get statement?